tools/convert_datasets/potsdam.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+import os.path as osp
+import tempfile
+import zipfile
+import glob
+from skimage.io import imread, imsave
+import numpy as np
+import math
+import mmcv
+from tqdm import tqdm
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert potsdam dataset to mmsegmentation format')
+    parser.add_argument('dataset_path', help='potsdam folder path')
+    parser.add_argument('--tmp_dir', help='path of the temporary directory')
+    parser.add_argument('-o', '--out_dir', help='output path')
+    parser.add_argument('--clip_size', type=int, help='potsdam clip size', default=512)
+    parser.add_argument('--stride_size', type=int, help='potsdam overlap size', default=256)
```

Comment:
It is better to add some comments for these two arguments (why and used for) since it is different from general semantic segmentation.